### ISSUE TITLE
Editable author information field missing from Fre side

### ISSUE THREAD

**Issue by lgontard** [ytgov/yukon-ca#781]
@leeomara On the French side of the Places record, the editable field for author information is missing. You can see who the author is but you cannot change it.

https://yukon.ca/fr/places/services-aux-victimes-de-whitehorse
<img width="289" alt="image" src="https://github.com/user-attachments/assets/1cd0cbe5-e988-40cd-932a-77d2fb1df05a" />

**Comment by leeomara** [comment]
@lgontard I think that between this and..
- #777 
- #773

we need to review what records of authorship
- tracked where?
- exposed where?
- used for what purpose?

**Comment by lgontard** [comment]
Yeah, looks like the editor experience needs to be examined. I was wondering if most of these issues were related to the French side, but it doesn't seem like it.

**Comment by w3creatives** [comment]
Please review the PRs: #835 and #836

**Comment by leeomara** [comment]
@lgontard the Author can't be changed in English either. In both languages you can see the author and edit the "Revision log message". I feel like this is not an issue that needs -- or can be -- fixed.

@w3creatives what change was made in response to this issue being reported, as part of #835 and #836? I'm concerned that we made changes that aren't needed.

**Comment by w3creatives** [comment]
In the Content language and translation settings page....Authored by and Authored on fields were not translatable. We enabled those options. Please see the screenshot attached.

/en/admin/config/regional/content-language

![Image](https://github.com/user-attachments/assets/000faee6-efb9-49a5-9d9f-24b56e25cfb2)

**Comment by lgontard** [comment]
In prod we can edit the author on the Eng and Fre side now. We wanted to be able to edit this so that when staff leave their position the page can be reassigned to their replacement.

<img width="425" alt="Image" src="https://github.com/user-attachments/assets/be096531-8f0a-45f6-a895-dd15814bc5b1" />

![Image](https://github.com/user-attachments/assets/41604894-0f4d-44ed-872c-8b7687e48335)

**Comment by lgontard** [comment]
FrenchTranslation was wanting to have access to that field. So, I think this is resolved.

**Comment by w3creatives** [comment]
Please use the following config files:

Location: web/config/sync

We will also do the regular merge request and this is only because the current production environment code is different from UAT.

[781.zip](https://github.com/user-attachments/files/18816294/781.zip)
